internal/server/stat/golden/get_stat_all/branch.json

```diff
@@ -3965,7 +3980,21 @@
                 "2023-02-23": 12083679,
                 "2023-02-24": 12085063,
                 "2023-02-25": 12085063,
-                "2023-02-26": 12085125
+                "2023-02-26": 12085125,
+                "2023-02-27": 12087218,
+                "2023-02-28": 12088153,
+                "2023-03-01": 12089197,
+                "2023-03-02": 12108673,
+                "2023-03-03": 12110014,
+                "2023-03-04": 12110014,
+                "2023-03-05": 12110014,
+                "2023-03-06": 12111875,
+                "2023-03-07": 12112791,
+                "2023-03-08": 12114454,
+                "2023-03-09": 12131977,
+                "2023-03-10": 12133065,
+                "2023-03-11": 12133065,
+                "2023-03-12": 12133065
               },
               "measurementMethod": "NYT_COVID19_GitHub",
               "importName": "NYT_COVID19",
```

Comment:
Covid is not an important thing now, shall we remove Covid from golden? @shifucun